server.js

```diff
@@ -72,8 +72,9 @@ app.get('/', function(req, res) {
 
 /* check username and password and send appropriate response */
 app.post('/login', function(req, res) {
-    if (req.session.user) {
+    if ('user' in req.session) {
```

Comment:
what does this do? I dont get the change your change

Reply:
when you log in and then go back a page to the 'log in' page and try entering user and password. The system breaks. This was a fix for that.

Reply:
change it to if (req.session)

Reply:
We need to check if there is a session

Reply:
session is actually there when you load the website

Reply:
session is actually there when you load the page